### PR TITLE
WFK2-347 Update Petclinic

### PR DESCRIPTION
**Pull request by joshuawilson**
There are several updates here.  First I ran a diff between the old PetClinic and the current one in Spring.  From that there were 2 types of updates: First the format and structure did not match, so that was fixed.  Second, they updated it to use Spring 4 so there were some changed to the libs and code.  

For testing in our environments I seperated the work into several commits so that you may compare format changes.  Then you can compare updates to libraries. 

Finally I updated it to use the WFK BOM with Spring 4. (until that is release you need to get that to test this)

When you are ready for it to be merged let me know and I will squash if you want.

### CODE REVIEW

File: spring-petclinic/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-petclinic-spring: PetClinic Example using Spring 3.1
+spring-petclinic: PetClinic Example using Spring 3.2
 ======================================================
 Author: Ken Krebs, Juergen Hoeller, Rob Harrop, Costin Leau, Sam Brannen, Scott Andrews
```
Comment (sgilda) [inline spring-petclinic/README.md]: Please add 2 spaces at the end of each metadata tag so it renders with a line break.

**Comment by sgilda** [comment]
Are the artifacts available in the developer repository: http://jboss-developer.github.io/temp-maven-repo/?

I get these build errors:
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]  
[ERROR]   The project org.jboss.quickstarts.wfk:jboss-spring-petclinic:2.4.0-build-SNAPSHOT (/home/sgilda/GitRepos/jboss-wfk-quickstarts/spring-petclinic/pom.xml) has 11 errors
[ERROR]     Non-resolvable import POM: Could not find artifact org.jboss.bom.wfk:jboss-javaee-6.0-with-spring:pom:2.5.0-SNAPSHOT @ line 414, column 16 -> [Help 2]
[ERROR]     Non-resolvable import POM: Could not find artifact org.jboss.bom.wfk:jboss-javaee-6.0-with-tools:pom:2.5.0-SNAPSHOT @ line 421, column 16 -> [Help 2]
[ERROR]     'dependencies.dependency.version' for org.springframework:spring-jdbc:jar is missing. @ line 184, column 15
[ERROR]     'dependencies.dependency.version' for org.springframework:spring-aop:jar is missing. @ line 188, column 15
[ERROR]     'dependencies.dependency.version' for org.springframework:spring-webmvc:jar is missing. @ line 192, column 15
[ERROR]     'dependencies.dependency.version' for org.springframework:spring-tx:jar is missing. @ line 196, column 15
[ERROR]     'dependencies.dependency.version' for org.springframework:spring-context-support:jar is missing. @ line 201, column 15
[ERROR]     'dependencies.dependency.version' for org.springframework:spring-orm:jar is missing. @ line 211, column 15
[ERROR]     'dependencies.dependency.version' for org.springframework:spring-oxm:jar is missing. @ line 215, column 15
[ERROR]     'dependencies.dependency.version' for org.springframework:spring-jms:jar is missing. @ line 225, column 15
[ERROR]     'dependencies.dependency.version' for org.springframework:spring-test:jar is missing. @ line 357, column 15

**Comment by sgilda** [comment]
Somehow, the old petclinic-spring folder is still there, which makes it confusing. Did you remove it from git?

**Comment by joshuawilson** [comment]
I have submitted another PR with the same fix only it has been squashed, so this line should be closed.

**Pull request closed without merging**